src/Symfony/Component/VarDumper/CHANGELOG.md

```diff
@@ -1,6 +1,13 @@
 CHANGELOG
 =========
 
+7.3
+---
+
+ * Add casters for `Dba\Connection`, `SQLite3Result`, `OpenSSLAsymmetricKey` and `OpenSSLCertificateSigningRequest`
```

Comment:
curl, gd

Reply:
The casters were already existing (it's just they are moved to a separate class), should they still appear in the changelog?